diff --git a/src/proto/CryptoGetAccountBalance.proto b/src/proto/CryptoGetAccountBalance.proto
index 42aa94e..25d43c6 100644
--- a/src/proto/CryptoGetAccountBalance.proto
+++ b/src/proto/CryptoGetAccountBalance.proto
@@ -19,5 +19,5 @@ message CryptoGetAccountBalanceQuery {
 message CryptoGetAccountBalanceResponse {
     ResponseHeader header = 1; //Standard response from node to client, including the requested fields: cost, or state proof, or both, or neither
     AccountID accountID = 2; // The account ID that is being described (this is useful with state proofs, for proving to a third party)
-    uint64 balance = 3; // The current balance, in tinybars
+    uint64 balance = 3 [jstype=JS_STRING]; // The current balance, in tinybars
 }
