diff --git a/src/proto/Transaction.proto b/src/proto/Transaction.proto
index 1b3343b3..6ca30839 100644
--- a/src/proto/Transaction.proto
+++ b/src/proto/Transaction.proto
@@ -16,10 +16,10 @@ import "TransactionBody.proto";
  * The SignatureList field is deprecated and succeeded by SignatureMap.
  */
 message Transaction {
+    SignatureList sigs = 2 [deprecated = true]; // The signatures on the body, to authorize the transaction; deprecated and to be succeeded by SignatureMap field
+    SignatureMap sigMap = 3; // The signatures on the body with the new format, to authorize the transaction
 	oneof bodyData {
 		TransactionBody body = 1 [deprecated = true]; // the body of the transaction, which needs to be signed
 	    bytes  bodyBytes = 4; // TransactionBody serialized into bytes , which needs to be signed
     }
-    SignatureList sigs = 2 [deprecated = true]; // The signatures on the body, to authorize the transaction; deprecated and to be succeeded by SignatureMap field
-    SignatureMap sigMap = 3; // The signatures on the body with the new format, to authorize the transaction
 }
